app/src/components/common/BalanceRow.vue

```diff
@@ -80,11 +80,14 @@
 
     <!-- endTime span for Polkadot undelegations -->
     <div
-      v-if="unstakingBalance"
+      v-if="unstakingBalance" 
       :key="balance.denom + '_endtime'"
       class="table-cell endtime"
     >
-      <span>
+      <span v-if="unstakeClaimable">
+        Claimable
+      </span>
```

Comment:
Shouldn't it rather be `Claimable on` or `Claimable starting on` ? More clear

Reply:
No, when this appears they are already claimable. It could be "claimable since" but I think that is not necessary info

Reply:
oh, I see. Got it wrong, alright